configuration/briefings/part_briefing_west.sqf

```diff
@@ -34,7 +34,7 @@ _cre = player createDiaryRecord ["diary", ["Credits","
 <br/>
 *** Insert mission credits here. ***
 <br/><br/>
-Made with F3 (http://www.ferstaberinde.com/f3/en/)
+Made with CAFE3 (https://github.com/CombinedArmsGaming/CAFE3/)
```

Comment:
I notice we are consistent with removing the last / at the end of URLs elsewhere in this framework 

Should it maybe be changed to (https://github.com/CombinedArmsGaming/CAFE3) to be consistent?

Reply:
Good catch, thank you.

Reply:
Thank you for entertaining my pedantry, you made it extremely difficult to spot anything

Reply:
😳